mappings/net/minecraft/entity/player/PlayerEntity.mapping

```diff
@@ -54,6 +64,9 @@ CLASS net/minecraft/class_1657 net/minecraft/entity/player/PlayerEntity
 	METHOD method_21823 shouldCancelInteraction ()Z
 	METHOD method_21824 shouldDismount ()Z
 	METHOD method_21825 clipAtLedge ()Z
+	METHOD method_23668 checkFallFlying ()Z
+	METHOD method_23669 startFallFlying ()V
+	METHOD method_23670 stopFallFlying ()V
```

Comment:
These should be `*ElytraFlying` because it specifically checks for elytras.

Reply:
Don't think it's elytra; moving while falling count as well (like there's a stat)

Reply:
`23668` only does anything `if (itemStack2.getItem() == Items.ELYTRA && ElytraItem.isUsable(itemStack2)) {`

Reply:
![](https://user-images.githubusercontent.com/6596629/75565636-fefe7e00-5a56-11ea-85f9-10bcc6182a0f.png)
Maybe it's only `method_23668` that's hardcoded for elytras 🤔

Reply:
That's correct. If it weren't for the enum, I would have called it "gliding", because that's essentially what it is...